### ISSUE TITLE
Replace "from now" suffix with "in" prefix

### ISSUE THREAD

**Issue by damienrambout** [FormatterKit/FormatterKit#234]
Hi,

I've been using FormatterKit for quite a long time and I've always wondered why it says `2 days from now` instead of `in 2 days`. I'm not a native English speaker, but `in 2 days` seems much lighter than `2 days from now`.
Is `in 2 days` grammatically incorrect compared to `2 days from now`? Or could it be considered to replace `%@ from now` suffix with `in %@` prefix?

------------

Now that I look at the implementation details, I see the following localised string : 
```
/* Deictic Expression Format (#{Time} #{Ago/From Now} */
"Deictic Expression Format String" = "%1$@ %2$@";
```

This could explain why `from now` is used, because it **MUST** be a suffix. Inverted `%1$@` and `%2$@` would then yield strings such as `ago 2 days` 😅

A solution would be to have 2 separate localisation keys : one for the past, and one for the future :
```
/* Deictic Expression Format (#{Time} #{Ago}) */
"Past Deictic Expression Format String" = "%1$@ %2$@";
/* Deictic Expression Format (#{In} #{Time}) */
"Future Deictic Expression Format String" = "%2$@ %1$@";
```

Ideally, we would keep the localisation key `from now` and update its English translation:
```
/* Future Deictic Expression */
"from now" = "in";
```
Using a specific order for `%1$@` and `%2$@` for past and future would probably ease implementation updates. More importantly, it would allow to keep other languages untouched (i.e. keeping their current respective order of `%1$@ %2$@`).

What do you think?

**Comment by yas375** [comment]
I like the idea. Having future and past deictic expressions makes sense to me. I'm also not a native English speaker, but I know it would make sense for Russian too.

> "2 days ago" = "2 дня назад"
> "2 days from now" or "in 2 days" = "через 2 дня"

Let me double-check with some native english speakers what do they think about "in 2 days" vs "2 days from now".

Thanks!

**Comment by damienrambout** [comment]
@yas375 Any update about this issue?

**Comment by yas375** [comment]
@damienrambout sorry for the delay. One of native speakers I've asked about this told me that `1 hour from now` is more formal and `in 1 hour` more conversational. And shared examples:

> I'll be in Seattle in 3 weeks
> The first launch of the Space X rocket will take place 7 months from now.

I'm absolutely up for separating future and past deictic expression formats and I'm fine with using `in 1 hour` for English instead of `1 hour from now`.

I'm willing to review your PR more carefully and merge it this week. Also will update at least Russian localization to use.

Thanks for your work 👍

**Comment by mattt** [comment]
Fixed by #235. Thanks again, @damienrambout!
